package-lock.json

```diff
@@ -26,7 +26,7 @@
     "ansi-colors": {
       "version": "1.0.1",
       "resolved": "https://registry.npmjs.org/ansi-colors/-/ansi-colors-1.0.1.tgz",
-      "integrity": "sha512-yopkAU0ZD/WQ56Tms3xLn6jRuX3SyUMAVi0FdmDIbmmnHW3jHiI1sQFdUl3gfVddjnrsP3Y6ywFKvCRopvoVIA==",
+      "integrity": "sha1-6UxsMGAFr4tIIkAkHi896kuFX/M=",
```

Comment:
This file comes up as changed after the `npm install` for me too, and needs updating to the latest version (posting the linux version of the file here since I suspect getting a vanilla `package-lock.json` on macosx is pretty much impossible due to the platform dependent module being installed):

[package-lock.zip](https://github.com/betaflight/blackbox-log-viewer/files/1738629/package-lock.zip)